src/webapp/components/ClusterMap.js

```diff
@@ -31,13 +31,14 @@ const makeBounds = (features) => {
 
 const ClusterMap = ({ geoJsonData, containerStyle = {} }) => {
   const requestCode = getRequestParam();
-  const [showDrivingClusters, setShowDrivingClusters] = useState(false);
+  const [showDrivingRequests, setShowDrivingRequests] = useState(true);
+  const [showRegularRequests, setShowRegularRequests] = useState(true);
 
   let paramRequest;
   const { requests, drivingClusterRequests } = geoJsonData;
   const { features: reqFeatures } = requests;
   const { features: clusterFeatures } = drivingClusterRequests;
-  const allRequests = showDrivingClusters
+  const allRequests = showDrivingRequests
```

Comment:
I didn't mess with this logic, and I think that's okay, but wanted to flag it in case I'm missing something.